backend/src/types/DataTypes.d.ts

```diff
@@ -101,15 +101,16 @@ export type DBCandidateDeciderInstance = {
   isOpen: boolean;
 };
 
-export type DevPortfolio = {
+export type DBDevPortfolio = {
   name: string;
-  deadline: number;
-  earliestValidDate: number;
-  submissions: DevPortfolioSubmission[];
+  deadline: string;
+  earliestValidDate: string;
+  submissions: DBDevPortfolioSubmission[];
+  readonly uuid: string;
 };
 
-export type DevPortfolioSubmission = {
-  member: IdolMember;
+export type DBDevPortfolioSubmission = {
+  member: firestore.DocumentReference;
   openedPRs: string[];
   reviewedPRs: string[];
   status: 'valid' | 'invalid' | 'pending';
```

Comment:
@henry-li-06 What do you think about giving a status for each PR instead of the overall submission? My thought is that in `githubUtil`, we give a reason if an individual PR is invalid, but then we lose that information when we make the overall status just `valid` or `invalid`. It would also allow the frontend to display the validation result for each PR submitted. 

It would be relatively simple to derive the overall status from these results:
```
results.some((result) => result.status === 'valid');
```

Reply:
That could be a good idea, I think we'd need to modify the types and `validateSubmission` function for us to be able to do that.

Reply:
I'll make a PR this week for that

Reply:
@JacksonStaniec Could you make those changes in a separate PR for now. The changes in this PR can be merged into your PR whenever this PR gets merged in.